src/components/30-organisms/testimonials/index.js

```diff
@@ -38,8 +38,8 @@ class AXATestimonials extends InlineStyles {
     super();
 
     // default values props
-    this.title = '';
-    this.subtitle = '';
+    this.title = null;
```

Comment:
Why from type String to Type Object? I would revert this change. `title` and `subtitle` are set as strings in the props so please use as default value empty string and not an object.